tests/manual-tests/framework/preferences/fullPage/index.html

```diff
@@ -87,6 +87,7 @@
                     </li>
                     <li>Use the "save and apply" button to apply the preview styles to Preferences Editor</li>
                     <li>Use the "reset" button to reset the styles</li>
+                    <li>Be sure to test with the <a href="index-jQueryUI.html">jQuery UI widgets version as well.</a></li>
```

Comment:
I could also rename the `index.html` in each directory to `index-nativeHTML.html`, so that someone doing the tests would see more obviously there were two testing files. This would require the least reorganization of the file structure.